CHANGELOG.md

```diff
@@ -98,6 +98,8 @@ Main (unreleased)
 
 - Fixed a bug where UDP syslog messages were never processed (@joshuapare)
 
+- Add missing lock in `loki.write#Run` routine. (@thepalbi)
```

Comment:
Torn on including this, maybe something like "Updating configuration for loki.write no longer drops logs"? I